01_fib/baijie/fib_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := strconv.Quote("1\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main(), expected %v, actual %v", expected, actual)
+	}
+}
+
+func TestFib(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	type test struct {
+		input    int
```

Comment:
Unneeded white spaces between input and int.

Reply:
The white spaces were added by gofmt -w
"golangci-lint run" check failed if I don't perform gofmt.